doc/user/content/sql/alter-cluster.md

```diff
@@ -15,8 +15,22 @@ cluster, use [`ALTER ... RENAME`](/sql/alter-rename/).
 
 {{< diagram "alter-cluster-reset.svg" >}}
 
+### `with_options`
+
+{{< private-preview />}}
+
+{{< diagram "with-options-alter-cluster-set.svg" >}}
+
 {{% cluster-options %}}
 
+
+### `WITH (WAIT UNTIL READY)` options
+Field                         | Value      | Description
+------------------------------|------------|-------------------------------------
+**TIMEOUT**                   | `duration` | The maximum duration to wait for the new replicas to be ready.
+**ON TIMEOUT**                | `text`     | The action to take on timeout `COMMIT|ROLLBACK`. ROLLBACK will result in a timeout error.
```

Comment:
```suggestion
**ON TIMEOUT**                | `text`     | The action to take on timeout `COMMIT\|ROLLBACK`. `ROLLBACK` will result in a timeout error.
```

Q: what happens on commit if we time out waiting for the new replicas to be ready? Is it that it's the old ungraceful behavior ... such that there is downtime?

Reply:
Exactly! I've updated the description here. Let me know if adds the appropriate clarity.